modules/core_project_factory/main.tf

```diff
@@ -350,3 +350,12 @@ resource "google_project_service" "enable_access_context_manager" {
   project = google_project.main.number
   service = "accesscontextmanager.googleapis.com"
 }
+
+/******************************************
+  Configure default Network Service Tier
+ *****************************************/
+resource "google_compute_project_default_network_tier" "default" {
+  count        = var.default_network_tier != "" ? 1 : 0
```

Comment:
I tried to follow the pattern in the rest of this module, this shouldn't do anything if the setting isn't set, otherwise it just passes through whatever the user defines. This seems like a safe way to implement this module, to avoid interfering with any existing installs

Reply:
Thanks, sounds good to me!